internal/pkg/service/buffer/store/model/mapper/mapper.go

```diff
@@ -0,0 +1,204 @@
+package mapper
```

Comment:
Tento pkg by som dal pod `.../api` adresar, kedze sa spolieha na vygenerovany kod, ktory je tiez tam `/api/gen`.

Je to medziclanok medzi `model <-> api` a skor mi to sedi na tu API stranu.
Ak by si mal este napr. CLI, tak mapper do CLI stdout by som potom dal opat na tu `CLI` stranu .

Alebo este iny priklad, ked sa rozhodnes cele API zmazat, tak chces zmazat aj tento mapper, ... model moze existovat dalej pre ine ucely.